diff -uNr phpsysinfo-orig/index.php phpsysinfo/index.php
--- phpsysinfo-orig/index.php	2008-01-08 10:44:45.000000000 +0100
+++ phpsysinfo/index.php	2008-01-08 10:54:23.000000000 +0100
@@ -40,10 +40,10 @@
   $error->addError('extension_loaded(pcre)', 'phpsysinfo requires the pcre module for php to work', __LINE__, __FILE__);
 } 
 
-if (!file_exists(APP_ROOT . '/config.php')) {
+if (!file_exists('/etc/webapps/phpsysinfo/config.php')) {
   $error->addError('file_exists(config.php)', 'config.php does not exist in the phpsysinfo directory.', __LINE__, __FILE__);
 } else { 
-  require_once(APP_ROOT . '/config.php'); 			// get the config file
+  require_once('/etc/webapps/phpsysinfo/config.php'); 			// get the config file
 }
 
 if ( !empty( $sensor_program ) ) {
