--- phpsysinfo/includes/os/class.Linux.inc.php.orig	2005-01-22 16:10:45.168602016 +0100
+++ phpsysinfo/includes/os/class.Linux.inc.php	2005-01-22 16:07:15.758566823 +0100
@@ -452,6 +452,10 @@
       $buf = fgets($fd, 1024);
       fclose($fd);
       $result = 'Debian ' . trim($buf);
+   } elseif ($fd = fopen('/etc/pld-release', 'r')) {
+      $buf = fgets($fd, 1024);
+      fclose($fd);
+      $result = trim($buf);
    } elseif ($fd = fopen('/etc/SuSE-release', 'r')) {
       $buf = fgets($fd, 1024);
       fclose($fd);
@@ -497,6 +501,8 @@
   function distroicon () {   
    if (file_exists('/etc/debian_version')) {
       $result = 'Debian.gif';
+   } elseif (file_exists('/etc/pld-release')) {
+      $result = 'PLD.gif';
    } elseif (file_exists('/etc/SuSE-release')) {
       $result = 'Suse.gif';
    } elseif (file_exists('/etc/mandrake-release')) {
